namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-core-services-tooling/06-circleci-serviceaccount.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: circleci
+  namespace: probation-core-services-tooling
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: circleci
+  namespace: probation-core-services-tooling
+subjects:
+  - kind: ServiceAccount
+    name: circleci
+    namespace: probation-core-services-tooling
+roleRef:
+  kind: Role
```

Comment:
Should this be "ClusterRole" rather than "Role"?

Reply:
Yes. Thanks. I'll fix it

Reply:
Done